sites/default/sqlconf.php

```diff
@@ -9,7 +9,7 @@
 $host	= 'localhost';
 $port	= '3306';
 $login	= 'libreehr';
-$pass	= 'libreehr';
+$pass	= 'ngai';
```

Comment:
@Ngai-E sure you getting the latest codes? sqlconfig is not to be here :)

Reply:
No, sorry I was just testing something with Konikodes. I will close the pull request, it is not ready yet